.github/workflows/build-and-publish.yml

```diff
@@ -1,4 +1,5 @@
 name: Build and Publish
+run-name: Version 12.1.${{ github.run_number }} (${{ github.event.head_commit.message }})
```

Comment:
Unfortunately, it is not possible to include `${{env.APP_VERSION}}` (or anything else defined after the workflow start) here.
So need to duplictae the value